src/test/java/com/vinylteam/vinyl/dao/jdbc/JdbcVinylDaoITest.java

```diff
@@ -0,0 +1,410 @@
+            insertVinyls.setString(10, "cool release (Vinyl)");
+            insertVinyls.setString(11, "best artist");
+            insertVinyls.setString(12, "cool release (Vinyl) - best artist");
+            insertVinyls.setString(13, "pop");
+            insertVinyls.setString(14, "200 uah");
+            insertVinyls.setString(15, "https://vinylsite.com/there/coolreleasevinyl");
+            insertVinyls.setString(16, "https://imagestore.com/there/imagerelease.jpg");
+            insertVinyls.setInt(17, 2);
+            insertVinyls.setLong(18, 1);
+            insertVinyls.setString(19, "wow effect");
+            insertVinyls.setString(20, "best of the best");
+            insertVinyls.setString(21, "wow effect - best of the best");
+            insertVinyls.setString(22, "rock");
+            insertVinyls.setString(23, "1500 euro");
+            insertVinyls.setString(24, "https://bestvinyl.com/here/woweffect");
+            insertVinyls.setString(25, "https://imagestore.com/somewhere/wow.jpg");
+            insertVinyls.setInt(26, 1);
+            insertVinyls.setLong(27, 2);
+            insertVinyls.executeUpdate();
+        }
+        vinylList = new ArrayList<>();
+
+        Vinyl firstVinyl = new Vinyl();
+        firstVinyl.setVinylId(1);
+        firstVinyl.setRelease("cool release");
+        firstVinyl.setArtist("best artist");
+        firstVinyl.setFullNameVinyl("cool release - best artist");
+        firstVinyl.setGenre("pop");
+        firstVinyl.setPrice("55 uah");
+        firstVinyl.setVinylLink("https://vinylstore.com/somewhere/coolrelease");
+        firstVinyl.setImageLink("https://imagestore.com/somewhere/image.jpg");
+        firstVinyl.setShopId(1);
+        firstVinyl.setUniqueVinylId(1);
+        vinylList.add(firstVinyl);
+
+        Vinyl secondVinyl = new Vinyl();
+        secondVinyl.setVinylId(2);
+        secondVinyl.setRelease("wow effect");
+        secondVinyl.setArtist("best of the best");
+        secondVinyl.setFullNameVinyl("wow effect - best of the best");
+        secondVinyl.setGenre("rock");
+        secondVinyl.setPrice("1500 euro");
+        secondVinyl.setVinylLink("https://bestvinyl.com/here/woweffect");
+        secondVinyl.setImageLink("https://imagestore.com/somewhere/wow.jpg");
+        secondVinyl.setShopId(1);
+        secondVinyl.setUniqueVinylId(2);
+        vinylList.add(secondVinyl);
+
+    }
+
+    @AfterEach
+    void afterEach() throws SQLException {
+        try (Statement truncateUniqueVinyls = connection.createStatement();
+             Statement truncateVinyls = connection.createStatement()) {
+            truncateUniqueVinyls.executeUpdate(TRUNCATE_UNIQUE_VINYLS);
+            truncateVinyls.executeUpdate(TRUNCATE_VINYLS);
+        }
+    }
+
+    @Test
+    void saveUniqueVinylsITest() throws SQLException {
+        try (Statement truncateUniqueVinyls = connection.createStatement()) {
+            truncateUniqueVinyls.executeUpdate(TRUNCATE_UNIQUE_VINYLS);
```

Comment:
Why truncate when you truncate in beforeAll and afterEach.

Reply:
BeforeAll - because before running the tests, you need to have nothing in the tables
AfterEach - because you need to clear the tables every time after each test